samples/tabcmd/Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM openjdk:8
 
 ADD https://downloads.tableau.com/esdalt/2018.2.3/tableau-tabcmd-2018-2-3_all.deb /tableau-tabcmd-2018-2-3_all.deb
 
-RUN apt install ./tableau-tabcmd-2018-2-3_all.deb
-RUN ./opt/tableau/tabcmd/bin/tabcmd --accepteula
+RUN apt-get install ./tableau-tabcmd-2018-2-3_all.deb
+RUN ./opt/tableau/tabcmd/bin/tabcmd --accepteula > /dev/null
+
+ENV PATH="/opt/tableau/tabcmd/bin:${PATH}"
```

Comment:
Is this used anywhere currently? Was it something that had to be set manually by the user, and this is the default one?

Reply:
Previously you had to specify the full path to tabcmd